inc/thorvg.h

```diff
@@ -1754,7 +1756,7 @@ class TVG_API WgCanvas final : public Canvas
      * @see Canvas::viewport()
      * @see Canvas::sync()
      */
-    Result target(void* instance, void* surface, uint32_t w, uint32_t h) noexcept;
+    Result target(void* instance, void* adapter, void* device, void* surface, uint32_t w, uint32_t h) noexcept;
```

Comment:
Please double-check the `adapter` parameter here. I don't see any necessity, otherwise please remove it.
```
Result target(void* instance, void* surface, uint32_t w, uint32_t h, void* device = nullptr)
```